blackberggroup/VA-OAM
# Feat/jls 3540899406/add oam overview video

## What type of PR is this? (check all applicable)


## Description Add new OAM video to News and Events landing page under "Podcasts"
Change "Podcasts" title to "Videos and Podcasts" and add new "About OAM" item
Create new page for new About OAM video

Change name of "podcasts" directory to "videos-podcasts" and remove redundant "podcasts" directory and it's assets from _site

## Related Tickets & Documents
- Monday Task https://blackberg-group.monday.com/boards/1931719392/pulses/7641277612/posts/3540899406

## QA Instructions, Screenshots, Recordings
Ensure "Podcasts" title has been changed to "Videos and Podcasts" and now has two items: new "About" and already existing "3D printing".
Ensure images are displaying correctly
Make sure links to both go to their respective pages and paths now include ../videos-podcasts/

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _includes/news-and-events/videos-podcasts.html
 - _site/assets/img/news-and-events/videos-podcasts/about-oam.png
 - _site/news-and-events/podcasts/about-oam/index.html
 - _site/news-and-events/videos-podcasts/How-3D-Printing-is-Transforming-Health-Care/index.html
 - _site/news-and-events/videos-podcasts/about-oam/index.html
 - assets/img/news-and-events/videos-podcasts/about-oam.png
 - news-and-events/videos-podcasts/about-oam.html
 - _site/news-and-events/fy24-q2/assistive-technology-library-launch/index.html
 - _site/news-and-events/fy24-q2/radiotherapy-bolus/index.html
 - _site/news-and-events/fy24-q2/richmond-lab-cutting-ceremony/index.html
 - _site/news-and-events/fy24-q2/thermal-fuse-cover/index.html
 - _site/news-and-events/fy24-q2/veteran-prevails-with-assistive-technology/index.html
 - _site/news-and-events/fy24-q3/additive-manufacturing-dental-technician/index.html
 - _site/news-and-events/fy24-q3/dean-qms/index.html
 - _site/news-and-events/fy24-q3/naloxone-trainer/index.html
 - _site/news-and-events/fy24-q3/pre-procedural-medical-models/index.html
 - _site/news-and-events/fy24-q3/transforming-smiles-impact-of-innovative-denture-solutions-for-veterans/index.html
 - _site/news-and-events/index.html
 - news-and-events/fy24-q2/assistive-technology-library-launch.html
 - news-and-events/fy24-q2/radiotherapy-bolus.html
 - news-and-events/fy24-q2/richmond-lab-cutting-ceremony.html
 - news-and-events/fy24-q2/thermal-fuse-cover.html
 - news-and-events/fy24-q2/veteran-prevails-with-assistive-technology.html
 - news-and-events/fy24-q3/additive-manufacturing-dental-technician.html
 - news-and-events/fy24-q3/dean-qms.html
 - news-and-events/fy24-q3/naloxone-trainer.html
 - news-and-events/fy24-q3/pre-procedural-medical-models.html
 - news-and-events/fy24-q3/transforming-smiles-impact-of-innovative-denture-solutions-for-veterans.html
 - news-and-events/index.html
 - _site/assets/img/news-and-events/videos-podcasts/3dprining.jpg
 - _site/news-and-events/podcasts/how-3d-printing-is-transforming-health-care/index.html
 - assets/img/news-and-events/videos-podcasts/3dprining.jpg
 - news-and-events/videos-podcasts/how-3d-printing-is-transforming-health-care.html
